docs/src/tutorials/pbt-and-tla.md

```diff
@@ -0,0 +1,1000 @@
+This means that committing an approval is selected with a probability of \\(\frac{1}{6}\\) at each step, whereas \\(\text{approve}(u_1,u_2,v)\\) is selected with a probability of \\(\frac{1}{6  \cdot N_{addr}^2  \cdot N_{val}}\\), for any concrete choice of \\(u_1, u_2, v\\) (and a probability of \\(\frac{1}{6}\\) for _some_ choice of \\(u_1,u_2,v\\)).
+
+Under these constraints, what is then the probability of executing a trace, which leads to an invariant violation?
+
+Let us define the following events:
+  - \\(\omega\\): a sequence of action instances is chosen, such that the invariant is violated
+  - \\(\alpha\\): a sequence of actions is chosen, such that the invariant is violated for some selection of instance parameters
+
+Observe that \\(\alpha \cap \omega = \omega\\).
+
+We break the problem into two parts: Finding an action order, and finding parameters, assuming the order is fixed.
+Concretely, we are going to find \\(P(\alpha)\\) and \\(P(\omega \mid \alpha)\\). Then, by the conditional probability rule,
+\\[
+P(\omega) =  P(\alpha \cap \omega) = P(\omega \mid \alpha) \cdot P(\alpha)
+\\]
+
+### Action orders that may lead to an invariant violation
+
+In this section, we determine \\(P(\alpha)\\).
+
+To find an invariant violation, we need the following five actions
+  - \\(H, L\\) (both \\(\text{submit\_approve}\\), but distinguished based on which of them is referenced by the commit action, i.e. whether the value is high or low)
+  - \\(T\\) (\\(\text{submit_transferFrom\}\\)) 
+  - \\(C_A\\) (\\(\text{commit\_approve}\\) referencing \\(H\\))
+  - \\(C_T\\) (\\(\text{commit\_transferFrom}\\))
+
+under the following constraints:
+  - \\(H\\) precedes \\(C\\) in the order
+  - \\(C_T\\) is last in the sequence of 5 actions
+
+There are 12 valid sequences that satisfy the constraints:
+\\[
+HLTC_AC_T,
+HTLC_AC_T,
+THLC_AC_T,
+LHTC_AC_T
+\\]
+\\[
+LTHC_AC_T,
+TLHC_AC_T,
+HLC_ATC_T,
+HTC_ALC_T
+\\]
+\\[
+THC_ALC_T,
+LHC_ATC_T,
+HC_ALTC_T,
+HC_ATLC_T
+\\]
+
+This means that \\(P(\alpha)\\), the probability of finding the 5 transaction sorts in the order needed to violate the invariant is \\(\frac{12}{6^5}\\).
+
+### Parameter instantiations that may lead to an invariant violation
+
+In this section, we determine \\(P(\omega \mid \alpha )\\).
+
+Assume now that we have selected an action order, which may lead to an invariant violation. We need to instantiate the parameters to \\(H,L,T\\):
+  - \\(H: \text{submit\_approve}(sender_H, spender_H, v_H)\\)
+  - \\(L: \text{submit\_approve}(sender_L, spender_L, v_L)\\)
+  - \\(T: \text{submit\_transferFrom}(sender_T, from, to, v_T)\\)
+
+subject to the following constraints:
+  - \\(c_1: sender_H \ne spender_H\\)
+  - \\(c_2: sender_L \ne spender_L\\)
+  - \\(c_3: sender_T, from, to\\) pairwise distinct
+  - \\(c_4: sender_H = sender_L = from\\)
+  - \\(c_5: spender_H = spender_L = sender_T\\)
+  - \\(c_6: v_H \ge v_T > v_L > 1\\)
```

Comment:
Why `> 1`? It doesn't seem required, since
```
tx1: approve(me, 5)
tx2: approve(me, 1)
commit tx1
tx3: transferFrom(me, badGuy, 2)
commit tx3
```
violates the invariant

Reply:
1 because the original amounts are `0..(N-1)`, but in the model they are `1..N` (nicer formulas w.r.t bounds). So 1 plays the role of `0` in the real system

Reply:
Makes sense! I'd suggest reminding the reader of this in the text (maybe I overlooked it, but a sentence right beneath the constraints should suffice)